src/schema/event/resolvers.ts

```diff
@@ -1,7 +1,7 @@
-import Promise from 'promise';
-
 import composeQuery from '../../utils/composeQuery';
 import normalizeKeys from '../../utils/normalizeKeys';
+import { EventParams } from './types';
+import { buildEventQuery } from './utils';
 
 const eventDetailsQueryBuilder = (include: string[]) => {
```

Comment:
You could move this function also to utils file and use same naming convention as for the other query builder function